setup_flashgg.sh

```diff
@@ -47,6 +47,8 @@ git cms-merge-topic -u simonepigazzini:topic_flashgg_10_5_0_qgtagger
 
 # EGamma post reco tools for energy scales and smearings
 echo "Settinga up Scales and Smearings form EGM"
+git cms-addpkg RecoBTag/TensorFlow
```

Comment:
Can you please move this on other lines (not in between EGM stuff) with an explanation of why we need to cherrypick this particular commit (to help future bookkeeping). Thanks

Reply:
Added proper explanation of DeepJet new training for this commit